eng/azure-pipelines.yml

```diff
@@ -177,6 +181,8 @@ stages:
               archflag: --arch arm64
               LLVMTableGenArg: /p:LLVMTableGenPath=$(Build.SourcesDirectory)/artifacts/obj/BuildRoot-x64/bin/llvm-tblgen
               ClangTableGenArg: /p:ClangTableGenPath=$(Build.SourcesDirectory)/artifacts/obj/BuildRoot-x64/bin/clang-tblgen
+              ClangVersion: clang-9
+              ClangPlusVersion: clang++-9
```

Comment:
It seems to me this is a mistake because there were only set when BuildOS == Linux https://github.com/dotnet/llvm-project/pull/287/commits/5c7ee7af897fc6e0a951618ac842753052e60cc3

Reply:
And I missed the Linux x64 case